### PR TITLE
feat(@wallet): Enable sepolia network

### PR DESCRIPTION
**Pull request by alaibe**
status-go: https://github.com/status-im/status-go/pull/4092

### CODE REVIEW

**Review by stefandunca — APPROVED** [review]
🤩

File: ui/app/AppLayouts/Wallet/panels/WalletNftPreview.qml [hunk 1]
```diff
@@ -103,7 +104,14 @@ ColumnLayout {
                     radius: 8
                     visible: nftPreviewSensor.hovered && !!root.tokenId && !!root.tokenAddress
                     onClicked: {
-                        const link = areTestNetworksEnabled ? Constants.networkExplorerLinks.goerliEtherscan : Constants.networkExplorerLinks.etherscan
+                        let link = Constants.networkExplorerLinks.etherscan
```
[the remark this answers is not in the document]
Comment (alaibe) [reply ui/app/AppLayouts/Wallet/panels/WalletNftPreview.qml:107]: I'd prefer we use the explorer link from the model than constants

File: src/app_service/service/settings/service.nim [hunk 2]
```diff
@@ -484,6 +484,16 @@ QtObject:
       return true
     return false
 
+  proc isSepoliaEnabled*(self: Service): bool =
+    return self.settings.isSepoliaEnabled
+
+  proc toggleIsSepoliaEnabled*(self: Service): bool =
+    let newValue = not self.settings.isSepoliaEnabled
+    if(self.saveSetting(KEY_IS_SEPOLIA_ENABLED, newValue)):
+      self.settings.isSepoliaEnabled = newValue
```
Comment (saledjenic) [inline src/app_service/service/settings/service.nim:493]: Add this line:
`self.events.emit(SIGNAL_USE_OF_SEPOLIA_NETWORK_UPDATED, SettingsBoolValueArgs(value: self.settings. isSepoliaEnabled))`

File: src/app/modules/main/profile_section/wallet/networks/view.nim [hunk 3]
```diff
@@ -46,6 +47,24 @@ QtObject:
...
+    read = getIsSepoliaEnalbled
+    notify = isSepoliaEnabledChanged
+
+  proc setIsSepoliaEnabled*(self: View, isSepoliaEnabled: bool) =
+    self.isSepoliaEnabled = isSepoliaEnabled
+    self.isSepoliaEnabledChanged()
+
+  proc toggleIsSepoliaEnabled*(self: View) {.slot.} =
+    self.delegate.toggleIsSepoliaEnabled()
+    self.isSepoliaEnabled = not self.isSepoliaEnabled
```
Comment (saledjenic) [inline src/app/modules/main/profile_section/wallet/networks/view.nim:65]: @alaibe remove this and the line below, cause that will update the toggle on the UI regardless of the real change result.

Instead, you should try to connect to `SIGNAL_USE_OF_SEPOLIA_NETWORK_UPDATED` signal and update the UI based on the value of the `value` argument `self.view. setIsSepoliaEnabled(args.value)`

**Review by saledjenic — APPROVED** [review]
@alaibe small suggestion only.

**Pull request merged**